flink-kubernetes/src/main/java/org/apache/flink/kubernetes/cli/KubernetesSessionCli.java

```diff
@@ -81,7 +81,8 @@ public KubernetesSessionCli(Configuration configuration, ClusterClientServiceLoa
 
 	public Configuration getEffectiveConfiguration(String[] args) throws CliArgsException {
```

Comment:
This can now become package private and be annotated as `@VisibleForTesting` I think.

Reply:
👌